src/commands/send-request.js

```diff
@@ -112,7 +112,7 @@ const getVariables = async (args) => {
   return result;
 };
 
-const sendRequest = async (args) => {
+export const sendRequestCore = async (args, options = {}) => {
```

Comment:
I don't think introducing options is necessary. It seems it duplicates existing functionality that bases on args